ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/theme/DarkTheme.java

```diff
@@ -1865,4 +1890,99 @@ public String projectExplorerPlaceholderIconColor() {
   public String projectExplorerPlaceholderRowColor() {
     return "#333846";
   }
+
```

Comment:
a lot of new items for colors, i think we need to decrease color values

Reply:
done

Reply:
Could you please revert it and use theme defined constants? I'm worry text will become partially unreadable.

Reply:
Otherwise the workspace status panel can not be customized.

Reply:
We should use so many constants as we have. The best way - using only 5...6 custom colors for this widget (kind of Grey, Yellow, Blue, Green, Red).

done